ecl/eclcc/eclcc.cpp

```diff
@@ -2106,6 +2081,12 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
         {
             setDebugOption("syntaxCheck", tempBool);
         }
+        else if (iter.matchOption(optMaxErrors, "--maxErrors"))
+        {
+        }
+        else if (iter.matchOption(optUnsuppressImediateSyntaxErrors, "--unsuppressImediateSyntaxErrors"))
```

Comment:
Why was this option added??  It wasn't in the JIRA, and I'm not sure I see the benefit.